qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/VerifyLoginSteps.java

```diff
@@ -0,0 +1,103 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * 
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.automation;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import org.openmrs.contrib.qaframework.RunTest;
+
+public class VerifyLoginSteps extends Steps {
+
+    @Before(RunTest.HOOK.SELENIUM_DEFAULT_LOGIN)
+    public void visitHomePage() {
+        initiateWithLogin();
+    }
+    
+    @After(RunTest.HOOK.SELENIUM_DEFAULT_LOGIN)
+    public void destroy() {
+        quit();
+    }
+
+    @And("user verifies modules available on home page after login as super user")
+    public void verifyModulesAvailableOnHomePage() {
```

Comment:
From the method name, how do i tell the user role?

Reply:
Thanks, @dkayiwa for reviewing again. You won't tell the role from the method name. However the method that triggers the functionality gets initiated with `initiateLoginPage`.

Reply:
What is the difference between `super user` and `sys admin`?

Reply:
We have two login crendentials for the two. Super user logins in as admin, admin123 which is our default and we inherit them from https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/Steps.java#L71 basically the one we use for test.Properties
then for sys admin. he is entitled for login crendentials as sysadmin and sysadmin123 as seen here https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/helper/LoginPage.java#L41

Reply:
cc @dkayiwa .

Reply:
I find it confusing to have a sys admin without full privileges.

Reply:
Sys admin have full privileges. Its true and all apps can be accessed unless am missing out some apps that are accessed. For those in systemAdministration, we wont be able to access them via home page but instead systemadmin page

Reply:
If they are not on the home page, then why do we have them in these assertions? I thought your tests are about what buttons different roles should see on the home page.

Reply:
@sherrif10 did you see the above comment?

Reply:
ohh. sorry. For the comment above. Yes  tests are about what buttons different roles should see on the home page. we only have those assertion to false for those not accessible on home page and those assertion to true those accessible on home page. forexample if user logs in using sysadmin and sysAdmin123, you can only see two apps. on that due not , am using assertTrue( for the two apps). others not visible are false.

Reply:
Then why do we have assertions for buttons that do not appear on the home page?

Reply:
@dkayiwa  Different login credentials have different apps appearing on the home page.

Reply:
But i see you asserting for apps that will never appear on the home page regardless of who is logged in.